crypto/err/openssl.txt

```diff
@@ -1156,6 +1152,11 @@ PROV_F_AES_EINIT:109:aes_einit
 PROV_F_AES_INIT_KEY:110:aes_init_key
 PROV_F_AES_STREAM_UPDATE:111:aes_stream_update
 PROV_F_AES_T4_INIT_KEY:112:aes_t4_init_key
+PROV_F_BLAKE2_MAC_INIT:115:blake2_mac_init
```

Comment:
Why are we propagating these now?

Reply:
`util/mkerr.pl` made me do it.

Reply:
You know what they say.. A good workman never blames his tools.. :)